dataland-frontend/src/components/forms/parts/elements/derived/DriveMixFormElement.vue

```diff
@@ -15,6 +15,7 @@
             :is-required="false"
             v-model:percentageFieldValueBind="driveMixPerFleetSegmentInPercent"
             name="driveMixPerFleetSegmentInPercent"
+            validation="between:0,100"
```

Comment:
Is there a test to make sure we don't loose the validation again?

Reply:
Two tests written. one for drivemix the other for procurementcategory

Reply:
ok